client/latis.py

```diff
@@ -61,31 +75,46 @@ class Dataset:
     def __init__(self, latisInstance, name):
         self.latisInstance = latisInstance
         self.name = name
+        self.selection = []
```

Comment:
We should be able to keep all the operations in a single list so we can better preserve order of operation.